pkg/features/table.go

```diff
@@ -0,0 +1,44 @@
+/*
+Copyright 2021 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package features
+
+// Table structure for table-driven tests
+type Table []struct {
+	Name        string
+	Description string
+	Labels      Labels
+	Assessment  Func
```

Comment:
With this mode every test in the table is being turned into a feature. Would it make sense to retain the ability of being able to run multiple assessment for the feature intact in this mode as well ?

Reply:
@harshanarayana  That is a good point and I agree. The table represents a collection of assessment around the same feature.

Reply:
How about using Steps to implement the table-driven instead of introducing a new type (`tables`)?

Reply:
One benefit to table over feature is the ability to skip naming the table

Reply:
@helayoty You and @harshanarayana are correct, I will update the code to translate each table entry as a step (assessment) in the same feature.  I introduced a new type (table) to avoid overloading type `Step` ... the name table, in context of Go test, has special meaning and that's why I selected it.  But, definitely great input and will revisit it for sure.

Also there is an example already with this PR that shows how to use the